app/assets/javascripts/modules/help.js.coffee

```diff
@@ -61,30 +61,61 @@ ns = @edsc.models.page
                   <li>Processing Levels - raw, geophysical variables, grid, or model</li>
-                </ul></div></p>"
+                </ul></div></p>'
       element: "#master-overlay-parent"
       placement: 'right'
       showNext: true
       top: null
     }, {
+
       title: "Map Tools"
-      content: '<p>Use these standard map tools to configure and position the map as well as enable certain spatial search tools.</p>'
+      content: '<p>Use these map tools to configure and position the map as well as enable certain spatial tools.
+                  <table><tr>
+                    <td width="40%">
+                      <ul class="icon-list">
+                        <li class="leaflet-draw-list"><i class="leaflet-draw-edit-remove"></i> Delete a layer</li>
+                        <li class="leaflet-draw-list"><i class="leaflet-draw-edit-edit"></i> Edit a layer</li>
+                        <li class="leaflet-draw-list"><i class="leaflet-draw-draw-marker"></i> Search by spatial coordinate </li>
+                        <li class="leaflet-draw-list"><i class="leaflet-draw-draw-rectangle"></i> Search by spatial rectangle</li>
+                        <li class="leaflet-draw-list"><i class="leaflet-draw-draw-polygon"></i> Search by spatial polygon</li>
+                      </ul>
+                    </td>
+                    <td width="60%">
+                      <ul class="icon-list">
+                        <li class="projection-draw-list"><i class="projection-draw-arctic"></i> North Polar map projection</li>
+                        <li class="projection-draw-list"><i class="projection-draw-geographic"></i> Geographic map projection</li>
+                        <li class="projection-draw-list"><i class="projection-draw-antarctic"></i> South Polar map projection</li>
+                        <li class="zoom-draw-list"><i class="fa fa-fw fa-plus"></i> Zoom in </li>
+                        <li class="zoom-draw-list"><i class="fa fa-fw fa-home"></i> Reset zoom </li>
+                        <li class="zoom-draw-list"><i class="fa fa-fw fa-minus"></i> Zoom out </li>
+                        <li><i class="map-layers"></i> Map layers</li></ul>
+                    </td>
+                  </tr></table>
+                </p>'
       placement: 'top'
       element: '.projection-switcher-arctic'
       showNext: true
     }, {
       title: 'Toolbar'
-      content: '<p>Use the options available (upon logging in) in the application toolbar to view recent downloads, saved projects, and profile
-      information. You can provide feedback using our feedback module.</p>'
+      content: '<p>This is the end of the tour. Using this toolbar, you can:
+        <div class="tour-indent">
+          <ul>
+            <li class="toolbar-draw-list"><i class="fa fa-bullhorn"></i> Provide feedback</li>
+            <li class="toolbar-draw-list"><i class="fa fa-lock"></i> Login to Earthdata for more user tools</li>\
+          </ul></div></p>
```

Comment:
Let's move these closing element tags down to their own lines (and make sure they are indented the same as their opening tags.